assets/scss/local.scss

```diff
@@ -2,12 +2,6 @@
   min-height: 600px;
 }
 
-.govuk-main-wrapper .govuk-table a:visited,
```

Comment:
Removing this visited link style override in favour of adding the `govuk-link--no-visited-state` class to `<a>` tags as appropriate. Was getting a bit messy.